velodyne_driver/src/lib/input.cpp

```diff
@@ -126,6 +126,13 @@ InputSocket::InputSocket(
   my_addr.sin_port = htons(port);          // port in network byte order
   my_addr.sin_addr.s_addr = INADDR_ANY;    // automatically fill in my IP
 
+  // compatibility with Spot Core EAP, reuse port 2368
+  int val = 1;
+  if (setsockopt(sockfd_, SOL_SOCKET, SO_REUSEADDR, &val, sizeof(val)) == -1) {
+    perror("socketopt");
```

Comment:
Instead of using perror here, I think we should use `RCLCPP_ERROR` like we do elsewhere.  So maybe something more like:

```suggestion
    RCLCPP_ERROR(private_nh->get_logger(), "Error setting REUSEADDR: %s", ::strerror(errno));
```

Reply:
@clalancette accepted your suggestion at https://github.com/ros-drivers/velodyne/pull/442/commits/aa8220969f9887d8f219198cfadf057f751f0282.